test/example.js

```diff
@@ -1,3 +1,4 @@
 var languege = 'en' // typo
 console.log(languege) // typo
 console.log('welcom') // typo
+console.log('welcom') // typo
```

Comment:
```suggestion
console.log('welcome') // typo
```
Replace `welcom` with `welcome`.